sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeContentWithSelectionMarks.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.FormPage;
+import com.azure.ai.formrecognizer.models.FormRecognizerOperationResult;
+import com.azure.ai.formrecognizer.models.FormSelectionMark;
+import com.azure.ai.formrecognizer.models.FormTable;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.polling.SyncPoller;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.util.List;
+
+public class RecognizeContentWithSelectionMarks {
+    /**
+     * Main method to invoke this demo.
+     *
+     * @param args Unused. Arguments to the program.
+     *
+     * @throws IOException Exception thrown when there is an error in reading all the bytes from the File.
+     */
+    public static void main(final String[] args) throws IOException {
+        // Instantiate a client that will be used to call the service.
+        FormRecognizerClient client = new FormRecognizerClientBuilder()
+                                          .credential(new AzureKeyCredential(Configuration.getGlobalConfiguration().get("AZURE_FORM_RECOGNIZER_API_KEY")))
+                                          .endpoint(Configuration.getGlobalConfiguration().get("AZURE_FORM_RECOGNIZER_ENDPOINT"))
+                                          .buildClient();
```

Comment:
Let's keep the env vars out and samples consistent. Could think about this update for all or none in upcoming PR's/